doc/btp_gmcs.txt

```diff
@@ -0,0 +1,121 @@
+Generic Media Control Service(ID 23)
+=================================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read supported commands
+		Controller Index:       <controller id>
+		Command parameters:     <none>
+		Response parameters:    <supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Command Send
+
+		Controller Index:	<controller id>
+		Command parameters:	Opcode (1 octets)
+					Use Param (1 octet)
+					Param (4 octets)
+		Response parameters:    <none>
+
+		This command is used to write to Media Control Point characteristic
+		to perform operation defined by opcode.
+
+		Opcodes: 0x01 - Play
+			 0x02 - Pause
+			 0x03 - Fast Rewind
+			 0x04 - Fast Forward
+			 0x05 - Stop
+			 0x10 - Move Relative (uses Param)
+			 0x20 - Previous Segment
+			 0x21 - Next Segment
+			 0x22 - First Segment
+			 0x23 - Last Segment
+			 0x24 - Goto Segment (uses Param)
+			 0x30 - Previous Track
+			 0x30 - Previous Track
+			 0x31 - Next Track
+			 0x32 - First Track
+			 0x33 - Last Track
+			 0x34 - Goto Track (uses Param)
+			 0x40 - Previous Group
+			 0x41 - Next Group
+			 0x42 - First Group
+			 0x43 - Last Group
+			 0x44 - Goto Group (uses Param)
+
+		During operation, the IUT may send event:
```

Comment:
if those actions don't require any over the air GATT operations lets not used extra events but return what is needed in command reply